data/globalevents/scripts/serversave.lua

```diff
@@ -1,4 +1,4 @@
-local function ServerSave()
+function ServerSave()
```

Comment:
I doubt this change is related?

Reply:
It is actually, I re use this function in `data/events/scripts/game.lua` so I didn't have to write it twice